FreeRTOS-Plus/Demo/coreHTTP_Windows_Simulator/HTTP_S3_Download/README.md

```diff
@@ -146,17 +153,12 @@ Use the following command to attach the policy with the certificate you register
 aws iot attach-policy --policy-name Thing_Policy_Name --target <certificate-arn>
 ```
 
-#### 6. Request a security token:
-
-Make an HTTPS request to the credentials provider to fetch a security token. You have to supply the following information:
-
-Certificate and key pair: Because this is an HTTP request over TLS mutual authentication, you have to provide the certificate and the corresponding key pair to your client while making the request. Use the same certificate and key pair that you used during certificate registration with AWS IoT.
-
-RoleAlias: Provide the role alias (in this example, Thermostat-dynamodb-access-role-alias) to be assumed in the request.
+#### 6. Obtain the Credentials Provider Endpoint
 
-ThingName: Provide the thing name that you created earlier in the AWS IoT thing registry database. This is passed as a header with the name, x-amzn-iot-thingname. Note that the thing name is mandatory only if you have thing attributes as policy variables in AWS IoT or IAM policies.
```

Comment:
Why were you able to get rid of these steps?

Reply:
It was inaccurate, the only necessary part of this step was obtaining the credentials provider endpoint. See [this article](https://aws.amazon.com/blogs/security/how-to-eliminate-the-need-for-hardcoded-aws-credentials-in-devices-by-using-the-aws-iot-credentials-provider/) - the step was directly copied